server/data/manageAdjudicationsClient.ts

```diff
@@ -81,24 +79,19 @@ export default class ManageAdjudicationsClient {
 
   async getYourCompletedAdjudications(
     agencyId: string,
-    pageRequest: PageRequest
-  ): Promise<PageResponse<ReportedAdjudication>> {
-    const zeroBasedPageRequest = pageRequest.changeIndex(0)
-    const result = await this.restClient.get({
-      path: `/reported-adjudications/my/agency/${agencyId}?page=${zeroBasedPageRequest.number}&size=${zeroBasedPageRequest.size}`,
+    pageRequest: ApiPageRequest
+  ): Promise<ApiPageResponse<ReportedAdjudication>> {
+    return this.restClient.get({
+      path: `/reported-adjudications/my/agency/${agencyId}?page=${pageRequest.number}&size=${pageRequest.size}`,
     })
-    return plainToClassFromExist(new PageResponse<ReportedAdjudication>(0, 0, 0, null, 0), result).changeIndex(1)
   }
 
   async getAllCompletedAdjudications(
     agencyId: string,
-    pageRequest: PageRequest
-  ): Promise<PageResponse<ReportedAdjudication>> {
-    const zeroBasedPageRequest = pageRequest.changeIndex(0)
-    const result = await this.restClient.get({
-      path: `/reported-adjudications/agency/${agencyId}?page=${zeroBasedPageRequest.number}&size=${zeroBasedPageRequest.size}`,
+    pageRequest: ApiPageRequest
+  ): Promise<ApiPageResponse<ReportedAdjudication>> {
+    return this.restClient.get({
```

Comment:
The change in index is now done when converting to/from ApiPageRequest/Response

Reply:
Cool